### PR TITLE
Routine tooling upgrade

### PR DESCRIPTION
**Pull request by mokagio**
I'll soon start working on upgrading the Sentry library integration. Before that, I thought I'd get the tooling in as up-to-date a state as I could.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ steps:
       # that occurs the first time after cleaning if one of your dependences uses a binary framework
       # See https://bugs.swift.org/browse/SR-13803
 
-      build_and_test_pod 'ios test' || build_and_test_pod 'ios test'
+      build_and_test_pod ios test || build_and_test_pod ios test
```
Comment (AliSoftware) [inline .buildkite/pipeline.yml:24]: Good catch remembering to update those while updating from bash-cache `1.x` to `2.x` 👏

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-2.6.4
\ No newline at end of file
+2.7.4
```
Comment (AliSoftware) [inline .ruby-version:1]: 🎉

File: Gemfile.lock [hunk 3]
```diff
@@ -104,14 +106,17 @@ GEM
...
     faraday-patron (1.0.0)
     faraday-rack (1.0.0)
-    faraday_middleware (1.1.0)
+    faraday-retry (1.0.3)
+    faraday_middleware (1.2.0)
       faraday (~> 1.0)
-    fastimage (2.2.5)
-    fastlane (2.191.0)
+    fastimage (2.2.6)
+    fastlane (2.209.1)
```
Comment (AliSoftware) [inline Gemfile.lock:119]: 👌

**Review by AliSoftware — APPROVED** [review]
Nice! :shipit:

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (mokagio) [reply .buildkite/pipeline.yml:24]: Thanks! But really... CI failed, I didn't remember 🙃 But I'll take it anyway 💪

Comment (AliSoftware) [reply .buildkite/pipeline.yml:24]: 🤫 (I won't tell, promise 🤐) 😜

**Pull request merged**